src/components/Button.js

```diff
@@ -92,6 +92,8 @@ class Button extends React.PureComponent {
           }
 
           .drui-button--withTextAndIcon {
+            min-width: 124px;
```

Comment:
Why would we need this if this is a button with text?

Reply:
this is necessary for button that contains only a few letters to be at least 124px. @TMaszko suggested it

Reply:
Ok then